go.mod

```diff
@@ -3,17 +3,17 @@ module github.com/openstack-k8s-operators/keystone-operator
 go 1.17
 
 require (
-	github.com/go-logr/logr v1.2.0
-	github.com/gophercloud/gophercloud v0.24.0
+	github.com/go-logr/logr v1.2.3
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.17.0
 	github.com/openshift/api v3.9.0+incompatible
 	github.com/openstack-k8s-operators/lib-common v0.0.0-20220630111354-9f8383d4a2ea
+	github.com/openstack-k8s-operators/lib-common/pkg/openstack v0.0.0-00010101000000-000000000000
 	github.com/openstack-k8s-operators/mariadb-operator v0.0.0-20220516121356-119f8d825a71
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.23.6
-	k8s.io/apimachinery v0.23.6
-	k8s.io/client-go v0.23.6
+	k8s.io/api v0.24.2
+	k8s.io/apimachinery v0.24.2
+	k8s.io/client-go v0.24.2
```

Comment:
Did you mean to bump the k8s API packages in this changeset? Seems unrelated.

Reply:
ah no, not intentionally. I think the root cause is when I created the openstack module it got the 0.24.2 version and due to that import it also changed it here. let me return it there to 0.23.6 and we can update it later in a separate step

Reply:
done, reverted the deps so that we can stick with 0.23.6